### PR TITLE
add changelogs for 2015

### PR DESCRIPTION
**Pull request by nataliejeremy**
- Created new changelog content type  
- Added changelogs for 2015

@bmackinney @rachelwhitton I need your help with:
- Removing the date on the changelog
- Generating a list of all changelogs 

This closes issue #535

### CODE REVIEW

File: source/docs/changelogs/2015/07.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+##Platform Improvements
+
+####Single Sign-On (SSO) Release  
+We added two new features to help security conscious organizations work with our platform:
+
+[**SAML Login**](https://pantheon.io/docs/articles/organizations/saml-for-orgs/): For organizations with SAML compatible single sign-on solutions, Pantheon allows authentication of users to the Dashboard via your own login system, giving you central administrative control over employee access to the platform.
+
+**Secure Runtime Access**: Pantheon protects all access to runtime resources including Drush, WP-CLI, or backing services like MySQL, behind the security of an encrypted [SSH tunnel](https://pantheon.io/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services/). This mode requires developers first establish a secure channel between their workstation and the Pantheon platform for all communication.
+
+####Workflows > Jobs with Upgraded Notifications
```
Comment (ari-gold) [inline source/docs/changelogs/2015/07.md]: This seems ambiguous:
Workflow is greater than jobs
or
Workflow moving to jobs

**Comment by nataliejeremy** [comment]
@rachelwhitton Can you figure out why it won't build the list of changelogs?

**Comment by rachelwhitton** [comment]
@nataliejeremy 
Fixed so changelogs dir is populated:
screen shot 2015-07-02 at 4 26 21 pm

**Comment by rachelwhitton** [comment]
@nataliejeremy we'll either need a landing page for 2015, or remove the dir and have the md files in the changelog dir - 2015 in breadcrumbs leads to a 404. @bmackinney 
screen shot 2015-07-02 at 4 30 18 pm

**Comment by bmackinney** [comment]
You may also be able to re-visit the permalink- based on the filename strategy we tried: `2015-07-July.md` @nataliejeremy.

**Comment by rachelwhitton** [comment]
Also, should we remove "changelog" from the md title? Seems obvious enough to just have months listed. e.g. 

## February
## March
## April
## May

etc.

**Comment by rachelwhitton** [comment]
@bmackinney I changed the filenames to the format you suggested

Screenshot: 
screen shot 2015-07-06 at 1 00 03 pm

This removed the 2015 dir. Permalink example: `/docs/changelogs/2015-02-February`

**Comment by rachelwhitton** [comment]
I added Changelogs as a category on our landing page - this removes Required Reading (which is linked within our most popular doc Getting Started). Here's a screenshot:
screen shot 2015-07-06 at 1 03 14 pm


Feedback wanted :)

**Comment by ari-gold** [comment]
Let's keep link to required reading on front page, please.

**Comment by rachelwhitton** [comment]
@ari-gold @bmackinney - any ideas on how to generate traffic to the changelogs? or is that not necessary? just curious how they’ll be utilized now that we’ve built them into the docs.

**Comment by rachelwhitton** [comment]
From @ari-gold: "@rachelwhitton: What about a row of three on top: Getting Started, Required Reading, Guides and a row of two underneath “All Articles, Changelogs"
maybe the second row wouldn’t have icons"

Screenshot:
screen shot 2015-07-06 at 1 50 16 pm

Thoughts? @bmackinney

**Comment by ari-gold** [comment]
Thanks for keeping the required reading in there. LGTM!

**Comment by bmackinney** [comment]
:+1:

**Pull request merged**